karma.conf.js

```diff
@@ -32,6 +32,14 @@ module.exports = function (config) {
     logLevel: config.LOG_INFO,
     autoWatch: true,
     browsers: ['ChromeHeadless'],
+    browserDisconnectTimeout: 10000,
+    browserDisconnectTolerance: 3,
+    browserNoActivityTimeout: 60000,
+    flags: [
```

Comment:
Brilliant you saved me here, i was about to raise this code changes as a part of making jenkins snl-frontend job stable to.   https://github.com/karma-runner/karma-chrome-launcher/issues/154#issuecomment-397737222

Reply:
all thanks go to @zielinskikamil as it's copied from him